modules/discount-code/config/config-json.js

```diff
@@ -26,10 +26,14 @@ module.exports = function configJSON(req) {
         // Fill in the host with the host that this is running on.
         // It must run under HTTPS
         url: `https://${req.headers.host}/modules/discount-code/execute`,
-        // The amount of time we want Journey Builder to wait for our
+        // The amount of time we want Journey Builder to wait before cancel the request. Default is 60000, Minimal is 1000
         timeout: 10000,
+        // how many retrys if the request failed with 5xx error or network error. default is 0
         retryCount: 3,
```

Comment:
For each property, can we add a comment with the allowed ranges (0-5 for example), and default value for each if not provided?

Reply:
it is in Readme file.